CHANGELOG.md

```diff
@@ -1,5 +1,14 @@
 # Changelog
 
+## 0.4.4
```

Comment:
Just curious why it is not `0.4.0`? Sorry if I asked a dumb question if it was already discussed.

Reply:
that's a great question! the version number is set up `0.4.4` because smaller version numbers were used in earlier `.vsix` builds already. The version number needs to be incremented so that those extensions installed from `.vsix` could get auto-updated.

Also thanks for pointing it out! This is indeed a versioning mistake, what we should do next time is to use `0.x.0-dev` for local dev builds.

Reply:
I will be adding documentation on how semver versioning should be done; for the current release, the other way to work around this is to release as `0.5.0` (and our master branch's version number would become `0.6.0-dev`) and skip `0.4.0` instead

Reply:
Thank for the context. I think it is okay to keep `0.4.4` to avoid potential conflicts with any packages being published earlier unexpectedly.

Reply:
thanks for the reminder! I checked the commit history again, we updated the version number to `0.4.4` in https://github.com/ms-iot/vscode-ros/commit/3396af6f586462cd5b230cdf34e7384bd9165645 and there is another change https://github.com/ms-iot/vscode-ros/commit/d2603c4eb9f332a3b5a129f5351d7890d447ba0b after the version number update. I will be updating this pull request to update the release version to `0.4.5` instead.